make.ps1

```diff
@@ -0,0 +1,138 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Position=1)]
+    [String] $target = "release",
+    [String] $configuration = "Release",
+    [String[]] $frameworks=@('net45','netcoreapp2.0')
+)
+
+$global:Result = 0
+
+$_BASEDIR = Split-Path -Path $MyInvocation.MyCommand.Definition -Parent
+
+$_VSWHERE = [System.IO.Path]::Combine(${env:ProgramFiles(x86)}, 'Microsoft Visual Studio\Installer\vswhere.exe')
+$_VSINSTPATH = ''
+
+if([System.IO.File]::Exists($_VSWHERE)) {
+    $_VSINSTPATH = & "$_VSWHERE" -latest -requires Microsoft.Component.MSBuild -property installationPath
+} else {
+    Write-Error "Visual Studio 2017 15.2 or later is required"
```

Comment:
Should this be 15.3?

Reply:
While we may required 15.3 to build for .NET Core the vswhere check will pass with 15.2 or later.